scapy/tools/UTscapy.py

```diff
@@ -963,21 +989,21 @@ def main():
         if six.PY2:
             KW_KO.append("python3_only")
             if VERB > 2:
-                print("### Python 2 mode ###")
+                print("### Python 2 mode ###", file=sys.stderr)
```

Comment:
Bei Python 2 gibt es das `file` Argument noch nicht:

```
>>> print('test', file=sys.stderr)
  File "<stdin>", line 1
    print('test', file=sys.stderr)
                      ^
SyntaxError: invalid syntax
```

Reply:
```from __future__ import print_function``` does this.